test/client/native_delete_by_query_test.rb

```diff
@@ -72,6 +72,11 @@
         end
       end
 
+      it "removes :read_timeout param from native delete-by-query calls, since ES5 removes it" do
+        dbq = Elastomer::Client::NativeDeleteByQuery.new(@index.client, {}, read_timeout: "2m")
+        assert_equal dbq.parameters.key?(:read_timeout) || dbq.parameters.key?("read_timeout"), false
```

Comment:
IMHO the arguments should be reversed, because expected is supposed to go first: http://docs.seattlerb.org/minitest/Minitest/Assertions.html#method-i-assert_equal

Reply:
ah thanks, will update this